src/inset_state/project.cpp

```diff
@@ -108,14 +109,19 @@ Point interpolate_point_with_barycentric_coordinates(
     bary_x * v1_proj.y() + bary_y * v2_proj.y() + bary_z * v3_proj.y()};
 }
 
-void InsetState::project_with_delaunay_t()
+void InsetState::project_with_delaunay_t(bool output_to_stdout)
 {
   std::function<Point(Point)> lambda_bary =
     [&dt = proj_qd_.dt,
      &proj_map = proj_qd_.triangle_transformation](Point p1) {
       return interpolate_point_with_barycentric_coordinates(p1, dt, proj_map);
     };
   transform_points(lambda_bary);
+
+  if (output_to_stdout) {
+    transform_points(lambda_bary, true);
+  }
```

Comment:
Good catch. It was missing.

Reply:
Actually, I think I have no idea what this does haha. I think you added this in the past supposedly.

Reply:
Explain this to me over Slack later.